<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountEditFormDialog</class>
 <widget class="QDialog" name="AccountEditFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Edit Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Account properties</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>310</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="accountIdLabel">
       <property name="text">
        <string>Account Id</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="accountIdLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="departmentLabel">
       <property name="text">
        <string>Department</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="departmentComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="userGroupLabel">
       <property name="text">
        <string>User group</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="userGroupComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="roleLabel">
       <property name="text">
        <string>Role</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="roleComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="accountEnabledLabel">
       <property name="text">
        <string>Account enabled</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="accountEnabledCheckBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="accountLockedLabel_2">
       <property name="text">
        <string>Account locked</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="accountLockedCheckBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="accountExpiredLabel">
       <property name="text">
        <string>Account expired</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="accountExpiredCheckBox"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="credentialsExpiredLabel">
       <property name="text">
        <string>Credentials expired</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="credentialsExpiredCheckBox"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="creatorLineEdit"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="creatorLabel">
       <property name="text">
        <string>Creator</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="creationTimeLineEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="creationTimeLabel">
       <property name="text">
        <string>Creation time</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="modifierLineEdit"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="modificationTimeLineEdit"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Modification time</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>User information</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>User name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="userNameLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="genderLabel">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ageLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="telNoLabel">
       <property name="text">
        <string>Tel No.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="telNoLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passwordAgainLabel">
       <property name="text">
        <string>Password again</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordAgainLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passwordHintLabel">
       <property name="text">
        <string>Password hint</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="passwordHintLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="ageLabel">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Male</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Female</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>passwordAgainLineEdit</tabstop>
  <tabstop>passwordHintLineEdit</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>ageLineEdit</tabstop>
  <tabstop>telNoLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>descriptionLineEdit</tabstop>
  <tabstop>departmentComboBox</tabstop>
  <tabstop>userGroupComboBox</tabstop>
  <tabstop>roleComboBox</tabstop>
  <tabstop>accountEnabledCheckBox</tabstop>
  <tabstop>accountLockedCheckBox</tabstop>
  <tabstop>accountExpiredCheckBox</tabstop>
  <tabstop>credentialsExpiredCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountEditFormDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountEditFormDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
